l1-contracts/contracts/state-transition/ValidatorTimelock.sol

```diff
@@ -52,9 +51,13 @@ contract ValidatorTimelock is IExecutor, Ownable2Step {
     /// @dev The delay between committing and executing batches.
     uint32 public executionDelay;
 
-    constructor(address _initialOwner, uint32 _executionDelay) {
+    /// @dev Era's chainID
+    uint256 immutable eraChainId;
```

Comment:
Immutable is more like constant rather than storage variable, shall we use `ERA_CHAIN_ID` naming instead?